src/components/CommentList/CommentList.tsx

```diff
@@ -1,3 +1,21 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
```

Comment:
Can be simplified to `import { CommentInfo } from '../CommentInfo';`

(index file is always a default export area from the folder and by having such export from index 
`export * from './CommentInfo';` your code can easily be shorthanded :)